src/response/chi0.jl

```diff
@@ -107,13 +107,14 @@ precondprep!(P::FunctionPreconditioner, ::Any) = P
 # Solves (1-P) (H-εn) (1-P) δψn = - (1-P) rhs
 # where 1-P is the projector on the orthogonal of ψk
 # n is used for the preconditioning with ψk[:,n] and the optional callback
+# /!\ for the solver to return the good output, εk_extra needs to be equal to
+# the diagonal of ψk_extra'Hk*ψk_extra (when not empty), which is the case by
+# default when using the extra eigenvalues
```

Comment:
This is also true for the eps_nk, right? Say something like "It is assumed (and not checked) that psi'Hpsi = Diagonal(eps)" (leaving implicit that psi must be understood as the union of the psik and psik_extra...)

Reply:
(what I mean is that eps_nk = psi_nk'Hkpsi_nk matters for the physics but nor for the algebra)

Reply:
Ah, right, sorry. Then yeah OK. Your formulation still says diag(psi'Hpsi) = eps, which is weaker than psi'Hpsi = Diagonal(eps).